splunk-quarkus/clowdapp.yaml

```diff
@@ -2,15 +2,15 @@
 apiVersion: v1
 kind: Template
 metadata:
-  name: eventing-splunk-quarkus
+  name: eventing-integrations
```

Comment:
I'm wondering whether we should keep this as is and for the new integration define new `ClowdApp`. Then we can migrate it as needed.